packages/bpk-theming/src/BpkThemeProvider.js

```diff
@@ -31,6 +31,9 @@ const uniq = (arr) => {
 };
 
 const createStyle = (theme, themeAttributes) => {
+  if (!theme) {
```

Comment:
You need to change `themeAttributesPropType` to take this into account as well. Otherwise it'll still throw a prop type warning